highlight importance of setActive() for sign-in and sign-up flows (#987)
clerk/clerk-docs: docs/references/javascript/sign-up/sign-up.mdx

This method initiates a new sign-up flow. It creates a new `SignUp` object and de-activates any existing `SignUp` that the client might already had in progress.

> What you pass to `params` depends on the [configuration](/docs/authentication/configuration/sign-up-sign-in-options) of the instance; choices on the instance settings affect which `params` options are available to use.

Author: ```suggestion
What you pass to `params` depends on the [configuration](/docs/authentication/configuration/sign-up-sign-in-options) of your application instance; choices on the instance settings affect which `params` options are available to use.

```

> The `create` method will return a promise of the new `SignUp` object. This sign-up might be complete if you supply the required fields in one step. However, this is not mandatory. Clerk's sign-up process provides great flexibility and allows users to easily create multi-step sign-up flows.

Author: ```suggestion
The `create()` method will return a promise of the new `SignUp` object. This sign-up might be complete if you supply the required fields in one step. However, this is not mandatory. Clerk's sign-up process provides great flexibility and allows users to easily create multi-step sign-up flows.

```

This method initiates a new sign-up flow. It creates a new `SignUp` object and de-activates any existing `SignUp` that the client might already had in progress.

> What you pass to `params` depends on the [configuration](/docs/authentication/configuration/sign-up-sign-in-options) of your application instance; choices on the instance settings affect which `params` options are available to use.

Author: ```suggestion
What you pass to `params` depends on the [configuration](/docs/authentication/configuration/sign-up-sign-in-options) of your application instance.

```

This method initiates a new sign-up flow. It creates a new `SignUp` object and de-activates any existing `SignUp` that the client might already had in progress.

> What you pass to `params` depends on the [configuration](/docs/authentication/configuration/sign-up-sign-in-options) of your application instance.

Reviewer: ```suggestion
What you pass to `params` depends on which [sign-in options](/docs/authentication/configuration/sign-up-sign-in-options) you have enabled in your Clerk application instance.
```
Just making it more explicit so they don't have to click through to understand what we mean.
